cura/Backups/Backup.py

```diff
@@ -29,7 +29,8 @@ class Backup:
     # Re-use translation catalog.
     catalog = i18nCatalog("cura")
 
-    def __init__(self, zip_file: bytes = None, meta_data: Dict[str, str] = None) -> None:
+    def __init__(self, application: "CuraApplication", zip_file: bytes = None, meta_data: Dict[str, str] = None) -> None:
+        self.application = application
```

Comment:
This should be private?

Reply:
I don't really have an opinion here because `zip_file` and `metadata` are already public.

Reply:
Those are used by the backups manager, application is (should) not (at least not via Backup).

Reply:
Done in c04c7654c